app/build.gradle

```diff
@@ -145,15 +145,13 @@ dependencies {
     compile 'com.google.android.gms:play-services-appstate:6.5.87'
 
     //Google
-    compile 'com.google.code.gson:gson:2.3'
+    compile 'com.google.code.gson:gson:2.3.1'
     compile 'com.google:volley:4.3'
-    compile 'com.google.http-client:google-http-client-gson:1.18.0-rc'
+    compile 'com.google.http-client:google-http-client-gson:1.19.0'
     compile 'com.google.apis:google-api-services-plus:v1-rev126-1.18.0-rc'
```

Comment:
I also wanted to delete it but it is used. When I upgrade it to version `1.19.0` it exceeds the 65K dex method count.